media/js/gamestate.js

```diff
@@ -31,4 +31,19 @@
         }
     });
 
+    jQuery('#breadcrumb-main a').on('click', function() {
```

Comment:
jQuery supports multiple selectors: https://api.jquery.com/multiple-selector/

Reply:
I did not know that, thanks!